docs/guides/oracle-spec/validator-exit-bus.md

```diff
@@ -188,7 +190,7 @@ To calculate **future rewards**, it's needed to [predict](https://github.com/lid
 
 1. Calculate latest exit epoch number and amount of validators that are exiting in this epoch
 2. If queue is empty - exit epoch will be calculated as `current epoch + MAX_SEED_LOOK AHEAD + 1`. **MAX_SEED_LOOKAHEAD** constant needs to mitigate some attacks, more details [here](https://eth2book.info/bellatrix/part3/config/preset/#max_seed_lookahead)
-3. Calculate **churn limit** - like a rate-limit on changes to the validator set. Minimum is 4 validators per epoch. And recalculates each `CHURN_LIMIT_QUOTIENT = 2**16`. For example when active validators reaches up to 327,680 amount, `churn limit` rises to 5, [spec](https://github.com/ethereum/consensus-specs/blob/master/specs/phase0/beacon-chain.md#get_validator_churn_limit)
+3. Calculate **churn limit** - like a rate-limit on changes to the validator set. Minimum is 4 validators with a balance of 32 ETH per epoch. The churn limit changes in increments of `EFFECTIVE_BALANCE_INCREMENT = 1 eth`. [spec](https://github.com/ethereum/consensus-specs/blob/dev/specs/electra/beacon-chain.md#new-get_balance_churn_limit)
```

Comment:
Let's remove validators count, since it could be not even 4 validators (if it's balance will be 2048 ETH)
smth like "minimum rate is N eth per epoch".
Perhaps we should update first line to "like a rate-limit on a balance change in the the validator set".